client/analysis/output.go

```diff
@@ -315,57 +315,62 @@ func PrintBrakemanOutput(containerOutput string) {
 }
 
 // PrintSafetyOutput will print Safety output.
-func PrintSafetyOutput(containerOutput string) {
+func PrintSafetyOutput(mongoDBcontainerOutput string) {
 
-	if strings.Contains(containerOutput, "ERROR_REQ_NOT_FOUND") {
-		color.Red("[HUSKYCI][X] huskyCI couldn't find any requirements file...\n")
-		color.Red("[HUSKYCI][*] Safety :(\n\n")
+	if mongoDBcontainerOutput == "No issues found." {
+		color.Green("[HUSKYCI][*] Safety :)\n\n")
 		return
 	}
 
-	// Safety might return a JSON with the "\" and "\"" characters, which needs to be sanitized to be unmarshalled correctly
-	sanitizateContainerOutput := strings.Replace(containerOutput, "\\", "\\\\", -1)
-	sanitizateContainerOutput2 := strings.Replace(sanitizateContainerOutput, "\\\"", "\\\\\"", -1)
-
-	// Safety container returns warnings and the json output in the same string, which need to be split
-	warningFound := strings.Contains(containerOutput, "Warning: unpinned requirement ")
-	if warningFound {
-		tmpcOutput := analysis.StringToLastLine(containerOutput)
-		warningOutput := analysis.GetAllLinesButLast(containerOutput)
-		containerOutput = tmpcOutput
-		for _, warning := range warningOutput {
-			color.Yellow("[HUSKYCI] [!]: %s", warning)
-		}
-	}
-
-	if containerOutput == "No issues found." && !warningFound {
-		color.Green("[HUSKYCI][*] Safety :)\n\n")
+	if mongoDBcontainerOutput == "Requirements not found or this project uses latest dependencies." {
```

Comment:
With current huskyCI code we can't infer yet if there were errors finding requirements files or the project being analyzed does not set the version of it's dependency.